lib/index.js

```diff
@@ -372,6 +394,28 @@ function opt (option) {
   }
 };
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function(){
```

Comment:
Further, if we only take an array of strings, then there's no ambiguity about whitespace, and we don't need to filter or map the input.